### PR TITLE
Refactor: refactor add-slots.js

### PR DESCRIPTION
**Pull request by lusarz**

### CODE REVIEW

File: packages/create-instance/add-slots.js [hunk 1]
```diff
@@ -1,60 +1,39 @@
...
-      throwError('the slots option does not support strings in PhantomJS. Please use Puppeteer, or pass a component.')
-    }
-    const domParser = new window.DOMParser()
-    const _document = domParser.parseFromString(slotValue, 'text/html')
-    const _slotValue = slotValue.trim()
-    if (_slotValue[0] === '<' && _slotValue[_slotValue.length - 1] === '>' && _document.body.childElementCount === 1) {
-      elem = vm.$createElement(compileToFunctions(slotValue))
-    } else {
-      const compiledResult = compileToFunctions(`<div>${slotValue}{{ }}</div>`)
-      const _staticRenderFns = vm._renderProxy.$options.staticRenderFns
```
Comment (lusarz) [inline packages/create-instance/add-slots.js:26]: I removed this 'replacing' of `staticRenderFns ` and didn't experienced unit-test failures - is it ok to remove, or I miss something ?

File: packages/shared/util.js [hunk 2]
```diff
@@ -21,3 +21,11 @@ export const capitalize = (str: string) => str.charAt(0).toUpperCase() + str.sli
  */
 const hyphenateRE = /\B([A-Z])/g
 export const hyphenate = (str: string) => str.replace(hyphenateRE, '-$1').toLowerCase()
+
+export function toArray (value: any) {
+  return Array.isArray(value) ? value : [value]
+}
+
+export function isObject (obj: mixed): boolean %checks {
```
Comment (lusarz) [inline packages/shared/util.js:29]: This method is [copied from Vue](https://github.com/vuejs/vue/blob/dev/src/shared/util.js#L41-L43)

File: packages/create-instance/add-slots.js (hunk 1 above)
Comment (eddyerburgh) [reply packages/create-instance/add-slots.js:26]: I don't think it's OK to remove, but we should add some tests to cover it. Can you write a test to check that a static slot is added correctly.

Comment (lusarz) [reply packages/create-instance/add-slots.js:26]: Thanks @eddyerburgh for information - I'll add appropriate tests for that

Comment (lusarz) [reply packages/create-instance/add-slots.js:26]: I moved back `staticRenderFns` related code in pull request. To be honest I don't know what is static slot - I'm not yet familiar with everything related with Vue.

**Review by eddyerburgh — APPROVED** [review]

**Pull request merged**